polygon-validator-signer-balance-bot/README.md

```diff
@@ -0,0 +1,27 @@
+# POLY01 - Validator Signer ETH Balance Bot
+
+## Description
+This bot monitors the ETH balance in the account [0xa8B52F02108AA5F4B675bDcC973760022D7C6020](https://etherscan.io/address/0xa8B52F02108AA5F4B675bDcC973760022D7C6020) and triggers an alert;
+- when account balance falls below the threshold (1 ETH)
+- when account balance recovers and raises above the threshold
+
+## Supported Chains
+- Ethereum
+
+
+## Alerts: Below Threshold
+- POLY-01
+  - Fired when the account balance is below  specified threshold
+  - Severity is always set to "info" 
+  - Type is always set to "info"
+  - Metadata fields:
+   - `balance`: monitored address ETH balance
+
+## Alerts: Above Threshold
+- POLY-02
+  - Fired when the account balance is above  specified threshold
+  - Severity is always set to "info" 
+  - Type is always set to "info"
+  - Metadata fields:
+   - `balance`: monitored address ETH balance
+
```

Comment:
```suggestion
## Alerts
- POLY-01
  - Fired when the account balance is below  specified threshold
  - Severity is always set to "info" 
  - Type is always set to "info"
  - Metadata fields:
   - `balance`: monitored address ETH balance
- POLY-02
  - Fired when the account balance is above  specified threshold
  - Severity is always set to "info" 
  - Type is always set to "info"
  - Metadata fields:
   - `balance`: monitored address ETH balance

```
While I understand why the alerts were split up, it's not really necessary and it can be enough by having the details be underneath their respective alert id.